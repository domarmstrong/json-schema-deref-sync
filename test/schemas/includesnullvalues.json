{
  "title": "object with falsy values and references",
  "isNull": null,
  "isFalse": false,
  "isZero": 0,
  "properties": {
    "nullRef": {
      "$ref": "#/isNull"
    },
    "falseRef": {
      "$ref": "#/isFalse"
    },
    "zeroRef": {
      "$ref": "#/isZero"
    },
    "otherTitle": {
      "$ref": "#/title"
    }
  }
}
