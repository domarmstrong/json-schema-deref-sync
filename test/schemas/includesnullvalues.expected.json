{
  "title": "object with falsy values and references",
  "isNull": null,
  "isFalse": false,
  "isZero": 0,
  "properties": {
    "nullRef": null,
    "falseRef": false,
    "zeroRef": 0,
    "otherTitle": "object with falsy values and references"
  }
}
